dwi_ml/data/creation/hdf5_creator.py

```diff
@@ -1,586 +0,0 @@
-        Parameters
-        ----------
-        bundles_path : pathlib.Path
-            Path to bundles folder.
-        dwi_ref : np.ndarray
-            Reference used to load and send the streamlines in voxel space.
-
-        Returns
-        -------
-        output_tractogram : StatefulTractogram
-            All streamlines in voxel space.
-        output_lengths : List[float]
-            The euclidean length of each streamline
-        """
-
-        with Timer("Processing streamlines", newline=True):
-
-            # Initialize
-            output_tractogram = None
-            output_lengths = []
-            n_original_streamlines = 0
-            if not self.bundles:
-                # If no bundles described in the json file, we will treat the files
-                # found in bundles as wholebrain tractograms
-                chosen_bundles_config = [HDF5BundleConfig(p.stem) for p in
-                                         bundles_path.glob('*')]
-                if len(chosen_bundles_config) == 0:
-                    raise ValueError("No bundles found in the boundles folder!")
-            else:
-                chosen_bundles_config = self.bundles
-            available_bundles = list(bundles_path.iterdir())
-
-            for bundle_config in chosen_bundles_config:
-                bundle, bundle_original_count = self._load_and_process_one_bundle(
-                    bundle_config, available_bundles, bundles_path, dwi_ref)
-                if bundle is None:
-                    continue
-
-                # Keep track of original count
-                n_original_streamlines += bundle_original_count
-
-                # Compute euclidean lengths
-                output_lengths.extend(length(bundle.streamlines))
-
-                # Add processed bundle to output tractogram
-                if output_tractogram is None:
-                    output_tractogram = bundle
-                else:
-                    # Validate that tractograms are in the same space
-                    # Function doesnt exist anymore but should not be necessary
-                    # if we use SFT.
-                    assert are_tractograms_in_same_space(output_tractogram,
-                                                         bundle),\
-                        "Inconsistent tractogram space: {}".format(bundle)
-                    output_tractogram.streamlines.extend(bundle.streamlines)
-
-            # Transfer the streamlines to the reference space before bringing them
-            # to VOX space. NOTE: This is done in case the streamlines were tracked
-            # in a different space than the provided dataset reference
-            if output_tractogram is None:
-                output_streamlines_rasmm = []
-            else:
-                output_streamlines_rasmm = output_tractogram.streamlines
-            output_tractogram = StatefulTractogram(output_streamlines_rasmm,
-                                                   dwi_ref,
```

Comment:
@ppoulin91 , I am not sure I understood correctly this part. You were forcing the space but now it is an option for the user so I had to change this. Can you check that the management of space is correct in the new script? Thanks!